lms/templates/certificates/_accomplishment-banner.html

```diff
@@ -46,14 +53,18 @@ <h3 class="sr-only">${_("Print or share your certificate:")}</h3>
                     % if share_settings:
                       <%
                        facebook_share_text = _("I completed the {course_title} course on {platform_name}.").format(course_title=accomplishment_course_title, platform_name=platform_name)
+                       twitter_share_text = _("I completed a course on {platform_name}. See my certificate here!").format(platform_name=platform_name)
```

Comment:
I'm not a big fan of any link language that says "here" in it.  Active language is preferred like "Take a look at my certificate."  It describes what will happen when they follow the link.  "See my certificate here."  begs the question "where is here? aren't we already here?"  

In the immortal words of Buckaroo Banzai "No matter where you go, there you are."